.github/workflows/test.yaml

```
@@ -0,0 +1,19 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+on: push
+name: Test
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+    - name: Install
+      run: npm install
+    - name: Test
+      run: npm run test
```

Call me old-fashioned but I like my newlines at the ends of files.
```suggestion
      run: npm run test

```